ibis/pandas/core.py

```diff
@@ -330,7 +380,7 @@ def execute_until_in_scope(
 
     # pass our computed arguments to this node's execute_node implementation
     data = [
-        new_scope[arg.op()] if hasattr(arg, 'op') else arg
+        get_scope(new_scope, arg.op()) if hasattr(arg, 'op') else arg
```

Comment:
Perhaps `get_scope_item` or `set_scope_item` is better

Reply:
OK I will change these signature